analyses/mutation_counts/02_mutation-count-plots.Rmd

```diff
@@ -0,0 +1,282 @@
+We like to keep our documents formatted with the philosophy of one sentence per line.
+This approach interacts well with git and GitHub, which record edits based on the line that changed.
+So keeping each sentence on a separate line makes it easier to see what has changed when editing a large block of text; 
+if we just put spaces between sentences within a paragraph, they would appear to git as a single line, and a change anywhere would look like a change to the whole paragraph.
+To separate paragraphs in markdown, we have to use a blank line, so separating sentences onto their own lines won't affect the final output.
+
+This notebook is an example of an analysis that might be performed using the output of the `01_count-gene-mutations.R` script (though with a lot of meta-commentary about the steps and choices made along the way).
+We are assuming that counting was performed for the low-grade astrocytoma (LGAT) and medulloblastoma MAF files for this notebook.
+If you would like, you can make a copy of this notebook, change the input files (and probably some text), and apply the same analysis to different data sets!
+
+## Setup
+
+### Load packages
+
+Loading the packages you plan to use frequently first is usually good practice. 
+However, we don't always have to load all of the packages we plan to use.
+In the code below, you will sometimes see that we use functions with the form `package::function()`. 
+This `::` notation tells R to use the function from a particular package, whether it has been loaded with `library()` or not.
+This can be more efficient if you are only using a single function from a package, and it has the additional benefit of providing extra context about which package provided a particular function, which can be easy to lose track of if many packages are loaded, or when using an unfamiliar package.
+
+The `::` notation can also help avoid errors when more than one package uses the same name for a function. 
+In that case, the most recently loaded function will be used, but that may not be what you want!
+One of the most commonly seen conflicts is from the `dplyr` package, which has some of the same function names as base R packages, as you can see in the warning messages printed when it loads below.
+
+In the `01_count-gene-mutations.R` script, we used the `dplyr::` prefix for all of the `dplyr` functions to avoid this conflict and provide the extra context.
+In this notebook, we load the `dplyr` package (despite the risk) with the knowledge that we will not be using the base R versions of the conflicting functions.
+We can also take comfort in the fact that we and others reading the code are likely to be familiar with the (commonly-used) `dplyr` package, and with `ggplot2`, which we will be using for our plots.
+
+
+
+```{r packages}
+library(dplyr) # data wrangling tools
+library(ggplot2) # the plotting package we will use
+library(magrittr) # the "pipe" (%>%) we will use to facilitate sequential operations
```

Comment:
Since `dplyr` imports this the `magrittr` load isn't strictly needed, but this approach also makes it more "verbose" that we're piping. I don't mind either way.